### PR TITLE
Add configuration for transifex

### PR DESCRIPTION
**Pull request by come-nc**
fix https://github.com/nextcloud/user_migration/issues/248

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

**Review by nickvergessen — CHANGES_REQUESTED** [review]
Missing .l10nignore

**Comment by come-nc** [comment]
https://github.com/nextcloud/docker-ci/issues/438

File: .tx/config [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[main]
```
Comment (nickvergessen) [inline .tx/config:1]: Fixed a copy paste error here

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**